CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+# [2.2.2] - 2019-09-25
```

Comment:
```suggestion
# [2.2.3] - 2019-10-04
```

Reply:
Thanks. I started typing it out, then decided to copy/paste. Forgot to change that part.